tools/integration_tests/util/setup/setup.go

```diff
@@ -148,6 +148,14 @@ func SetUpTestDir() error {
 		}
 		binFile = path.Join(TestDir(), "bin/gcsfuse")
 		sbinFile = path.Join(TestDir(), "sbin/mount.gcsfuse")
+
+		// mount.gcsfuse will find gcsfuse executable in mentioned locations.
+		// https://github.com/GoogleCloudPlatform/gcsfuse/blob/master/tools/mount_gcsfuse/find.go#L59
+		// Copying the executable to /usr/local/bin
+		err := operations.CopyDirWithRootPermission(binFile, "/usr/local/bin")
```

Comment:
how was it working all these days then?

Reply:
[persistent mounting](https://cloud.google.com/storage/docs/gcsfuse-mount) integration tests got merged yesterday only. Tests are getting fail because of that.